holoviews/plotting/bokeh/raster.py

```diff
@@ -15,6 +15,9 @@
 
 class RasterPlot(ColorbarPlot):
 
+    nodata = param.Integer(default=None, doc="""
+        Missing data (NaN) value for integer data""")
```

Comment:
```suggestion
        Optional missing-data value for integer data.
        If non-None, data with this value will be replaced with NaN so that it is transparent when plotted.""")
```

Reply:
Add this suggestion to the batch even though it isn't actually true and will need updating again: see my previous screenshot with `'NaN':'green'` where `NaN` *isn't* transparent.